scale/ingest/container.py

```diff
@@ -0,0 +1,9 @@
+"""Defines the fields for handling file systems in the local container volumes for ingest and Strike jobs"""
+from __future__ import unicode_literals
+
+import os
+
+from storage.container import SCALE_ROOT_PATH
+
+
+SCALE_INGEST_MOUNT_PATH = os.path.join(SCALE_ROOT_PATH, 'ingest_mount')
```

Comment:
They are more hard-coded than they are configuration. Also some of the "container" modules include functions. I like having them separated into a file specific to "in-container" stuff.